src/utils/apiError.ts

```diff
@@ -0,0 +1,93 @@
+import httpStatus from 'http-status'
+
+/**
+ * ErrorCode
+ */
+export enum ErrorCode {
```

Comment:
Is this for proper localization on Front end?

Reply:
yes